contributors/design-proposals/apiserver-build-in-admission-plugins.md

```diff
@@ -43,8 +43,8 @@ These plugins should be moved. One example is the Namespace Lifecycle plugin whi
 from creating resources in non-existent namespaces.
 
 *Note*:
-For loading in-cluster configuration [visit](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/examples/in-cluster/main.go#L30)
- For loading the configuration directly from a file [visit](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/examples/out-of-cluster/main.go)
+For loading in-cluster configuration [visit](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/examples/in-cluster-client-configuration/main.go#L30)
```

Comment:
I checked the new url, they're ok. But since the code may change in the  future, I think we can remove the line anchor `#L30`